invokeai/frontend/web/src/features/nodes/components/flow/nodes/common/NodeWrapper.tsx

```diff
@@ -71,6 +75,7 @@ const NodeWrapper = (props: NodeWrapperProps) => {
       transitionDuration="0.1s"
       cursor="grab"
       opacity={opacity}
+      boxShadow={isErrorNode ? '0 0 10px 6px #7F3A41' : undefined}
```

Comment:
Nit: Instead of a hex code, we can use a CSS variable. For example, 'error.500' resolves to `var(--invoke-colors-error-500)`.

There's also a hook:
```ts
const [error500] = useToken('colors', ['error.500']);
```

At runtime, `error500` will be the CSS variable as a string, so you could use it in the shadow: 
```ts
boxShadow={isErrorNode ? `0 0 10px 6px ${error500}` : undefined}
```